tests/v2/validation/steve/vai/vai_test.go

```diff
@@ -22,6 +22,7 @@ import (
 	"sync"
 	"testing"
 	"time"
+	vaiaction "github.com/rancher/rancher/tests/v2/actions/vai"
```

Comment:
nit: name `vaiaction` since we just call it `vai` in the secrets encryption test (and I can't unsee "vacation")

Reply:
I removed the changes made in actions for vai and instead used extensions directly.